docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -7,22 +7,24 @@
 
 {description}
 
-NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage. See xref:deploy:enterprise-licenses.adoc[].
+NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage.
+See xref:deploy:enterprise-licenses.adoc[].
 
-== What Can You Do with Tiered Storage?
+Tiered Storage allows you to work with datasets that are larger than memory using the same Hazelcast map API.
+It allows you to optimize infrastructure costs and performance by combining a fast, high performance in-memory layer and a lower-cost, high-capacity disk-based storage layer.
+With a larger data capacity per member, Tiered Storage can also reduce the number of deployed Hazelcast cluster members.
 
-Have fewer Hazelcast members for large maps. For example, for a map with 1 TB of data, you'll need
-ten Hazelcast members if all data is kept in memory and each member has 100 GB of data. Tiered Storage allows you to split the data between memory and disks, which requires fewer cluster members depending on how the data is distributed.
+Accessing data in the memory offers superior access speed, but memory is much more expensive than disks, while the modern NVMe SSDs offer performance that makes Tiered Storage a viable alternative to memory-only storage at a much lower total cost of ownership.
```

Comment:
Im not convinced this adds additional value to what has already been discussed in the previous paragraph.

Reply:
Removed in https://github.com/hazelcast/hz-docs/pull/1047/commits/85eb21450d0833a0e33da49a49d306678275a0ac.